Added NBFT table support

johnmeneghini opened the pull request:
Added support for parsing and printing the contents of the NBFT table (per NVMe-oF boot specification v1.0).

Signed-off-by: Stuart Hayes <stuart_hayes@dell.com>
Signed-off-by: John Meneghini <jmeneghi@redhat.com>
Signed-off-by: Martin Wilck <mwilck@suse.com>

stuarthayes committed:
Added NBFT table support

Added support for parsing and printing the contents
of the NBFT table (per NVMe-oF boot specification v1.0).

Signed-off-by: Stuart Hayes <stuart_hayes@dell.com>
Signed-off-by: John Meneghini <jmeneghi@redhat.com>
Signed-off-by: Martin Wilck <mwilck@suse.com>
Signed-off-by: Martin Belanger <martin.belanger@dell.com>

johnmeneghini committed:
timberland: change the libnvme submodule and update README

The Timberland-sig project is using a copy
of the nvme-cli and libnvme repositories until release.

This change sets the libnvme submodule to timberland-sig
and updates the README.md file with work flow instructions.

The default/feature branch is now name timberlandmaster.
This applies to both nvme-cli and libnvme.

Add instructions how to override system libnvme

Update libnvme.wrap to use timberlandmaster

Note: when merging upstream, remove this sha.

Signed-off-by: John Meneghini <jmeneghi@redhat.com>

mwilck commented:
Why is the target branch `timberlandmaster_v2.2.1` ? Shouldn't it simply be `master` ?

igaw commented:
Please do an official PR against upstream. I'd like to get more of the upstream folks involved.

johnmeneghini commented:
Closing this request.  Please see the upstream request:

https://github.com/linux-nvme/nvme-cli/pull/1791

Note: I have updated this timberland sig repository and rebased this branch on to v2.3.
